Components/Admin/PendingUsers.js

```diff
@@ -63,22 +81,124 @@ class PendingUsers extends Component {
       Alert.alert("User has succesfully been deleted");
   }
   
+  async edit(pendingUser) {
+    this.setState({
+      editable: !!(this.state.editable % 2) ? pendingUser.editable : pendingUser.editable + 1,
+      currentUser: pendingUser,
+      currentCommittee: pendingUser.Committee
+    })
+    Alert.alert("You may now edit the user")
+  }
+
+  async submit(pendingUser) {
```

Comment:
Not submitting the user to firebase.auth

Reply:
approved User gets added for the first time in the checkUsers() method in Login.js

Reply:
got it, it's working 👍